{
  "id": "MAL-2024-5908",
  "summary": "Malicious code in requesttest3 (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "modified": "2024-06-25T13:41:46Z",
  "published": "2024-06-25T13:41:46Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "modified_time": "2024-06-25T13:41:46Z",
        "import_time": "2024-06-28T02:50:32.547652904Z",
        "versions": [
          "1.0.1",
          "1.0.2"
        ],
        "id": "RLMA-2024-04709",
        "source": "reversing-labs",
        "sha256": "45fd3dbd5757dfedd23e335857d4a95478ab665792c7f6a87e454c85837b3954"
      }
    ]
  },
  "affected": [
    {
      "package": {
        "name": "requesttest3",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/requesttest3"
      },
      "versions": [
        "1.0.1",
        "1.0.2"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/requesttest3/MAL-2024-5908.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}